otter/models/cass.py

```diff
@@ -206,7 +206,7 @@ def serialize_json_data(data, ver):
 
 _cql_count_for_tenant = (
     'SELECT COUNT(*) FROM {cf} '
-    'WHERE "tenantId" = :tenantId;')
+    'WHERE "tenantId"=:tenantId {deleting};')
```

Comment:
sorry, I kept misunderstanding because I didn't realize different tables were being passed to this query. It's all good :)